src/components/MapboxGlMap.svelte

```diff
@@ -45,6 +47,22 @@
   // Show tile boundaries on the map as desired
   $: if (map) map.showTileBoundaries = showBoundaries;
 
+  // Resize the map when adding more maps and changing container size
+  $: {
+    if (map && currentNumberOfMaps !== numberOfMaps) {
```

Comment:
Is tracking `currentNumberOfMaps` necessary? Or could we just rely on `numberOfMaps` being used here in a reactive block?

Reply:
Good call!